config/services/dataplane_v1beta1_openstackdataplaneservice_swift.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: swift
+spec:
+  label: swift
+#  openStackAnsibleEERunnerImage: quay.io/cschwede/openstack-ansibleee-runner:test
```

Comment:
I think we can drop this

Reply:
Oh yes, sorry for that - jus fixed and pushed an updated commit.